ui/app/actions.js

```diff
@@ -264,6 +265,21 @@ function addNewKeyring (type, opts) {
   }
 }
 
+function importNewAccount (strategy, args) {
+  return (dispatch) => {
+    dispatch(actions.showLoadingIndication('This may take a while, be patient.'))
```

Comment:
Yup!  Although this message will be a blur unless a person is importing a geth/mist wallet, but we can't know when we initialize it.